content/ja/docs/reference/command-line-tools-reference/kube-apiserver.md

```diff
@@ -0,0 +1,2535 @@
+---
```

Comment:
This file seems written in Chinese, please translate into Japanese or copy from English

Reply:
changed to the English content.